lib/between_meals/knife.rb

```diff
@@ -73,6 +74,18 @@ def cookbook_upload_all
       exec!("#{@knife} cookbook upload -a -c #{@config}", @logger)
     end
 
+    def berks_cookbook_upload_all
+      @cookbook_dirs.each do |path|
+        Dir.chdir(path)  # http://stackoverflow.com/a/808285/830426
```

Comment:
You'll leave the CWD of the program in a cbdir when you're done... please don't.

Also, your `exec` already does the directory change, so this shouldn't be necessary at all